Unix/configure

```diff
@@ -2298,6 +2300,12 @@ then
     cp $rootrelative/etc/omigen.conf \$destdir/$sysconfdir/omigen.conf
 fi
 
+
+if test ! -f "\$destdir/$sysconfdir/omireg.conf"
```

Comment:
This is an odd syntax. How about what we usually do, something like:

```
if [ ! -f "\$destdir/$sysconfdir/omireg.conf" ]; then
```

(and delete next line)